spec/system/candidate_interface/candidate_signing_in_spec.rb

```diff
@@ -1,23 +1,53 @@
 require 'rails_helper'
 
 describe 'A candidate signing in' do
+  include TestHelpers::SignUp
+
   before do
     visit '/'
     click_on 'sign in'
 
     fill_in t('authentication.sign_in.email_address.label'), with: 'new_candidate@example.com'
   end
 
-  it 'sees the the sign in page' do
+  it 'sees the sign in page' do
     expect(page).to have_content 'Enter the email address you used to register with'
   end
 
+  context 'when the email does not exist' do
+    it 'sees the "Check your email" page' do
+      fill_in t('authentication.sign_in.email_address.label'), with: 'non_existent_candidate@example.com'
+      click_on t('authentication.sign_up.button')
+
+      # It redirect to the home
+      expect(page).to have_content 'Check your email'
```

Comment:
Perhaps we test that it redirects instead of the page content here?

Reply:
That is a good point but in this case, we are rendering the exact same page is both scenarios:
- email found (candidate already signed up)
- email NOT found
no redirection
that's to avoid giving any clue to a potential attacker.

anyway I am gonna add a test to check the URL of the page

Reply:
Thanks, I was confused by the comment.